source/SIL.AppBuilder.Portal.Frontend/src/ui/components/inputs/date-range/styles.scss

```diff
@@ -7,18 +7,17 @@
 
 }
 
-
-.input {
-  svg {
-    position: absolute !important;
-    right: 0;
-  }
-}
-
 .date-range-input-with-label {
   svg {
     pointer-events: none;
   }
+  .input {
```

Comment:
I had to adjust the filters on the project directory screen to use css-grid, which resulted in some dynamic positioning at certain screen sizes. I noticed that the svg/icons were not always correctly positioned, so this had to change.

for reference: this is the filter list:
![image](https://user-images.githubusercontent.com/199018/52592445-6037bd00-2e14-11e9-9969-35e947d2a73c.png)
![image](https://user-images.githubusercontent.com/199018/52592450-65950780-2e14-11e9-860e-eba79e132602.png)
![image](https://user-images.githubusercontent.com/199018/52592457-6a59bb80-2e14-11e9-8173-25a6dc134fa4.png)